CMakeLists.txt

```diff
@@ -472,6 +470,31 @@ if (BUILD_TESTING)
     file (GLOB TEST_LD_PRELOAD "tests/LD_PRELOAD/*.c")
     add_library(allocator_overrides SHARED ${TEST_LD_PRELOAD})
 
+    set(VALGRIND_DEFAULT " \
+    --leak-check=full \
+    --leak-resolution=high \
+    --trace-children=yes \
+    --run-libc-freeres=no \
+    -q --error-exitcode=123 \
+    --error-limit=no \
+    --num-callers=40 \
+    --undef-value-errors=no \
+    --suppressions=valgrind.suppressions")
+
+    set(VALGRIND_PENDANTIC "${VALGRIND_DEFAULT} --errors-for-leak-kinds=all")
+
+    # Add pendentic Valgrind check for Libcrypto is openssl-1.1.1.
```

Comment:
We generally want to write comments in response to the questions that you know people will ask. Here I think the question would be "why are we only running pedantic valgrind on OpenSSL 1.1.1? Why not all of them?"

You mention that we are incrementally adding things, but I think that sort of buries the lede.

I'd go for a wording more like
> "pedantic valgrind" will error on memory that is "Still Reachable". We only run this on OpenSSL 1.1.1 because there are hundreds of false positives in other libcryptos. 
> https://github.com/aws/s2n-tls/issues/3758